### PR TITLE
style: button disabled states, min-width, and improved doc

### PR DESCRIPTION
**Pull request by ArianneArcher**
https://flo-scss-flo-11285-btn--gqobdc.herokuapp.com/

FLO-11285

### CODE REVIEW

File: scss/_buttons.scss [hunk 1]
```diff
@@ -21,6 +21,7 @@
   border: $btn-border-width solid transparent;
   @include button-size($btn-padding-y, $btn-padding-x, $btn-font-size, $btn-line-height, $btn-border-radius);
   @include transition($btn-transition);
+  min-width: from-pixels(154);
```
Comment (bexux) [inline scss/_buttons.scss:24]: I would like to discuss this.

**Comment by bexux** [comment]
Disabled buttons should not have cursor: pointer, but cursor: default (I believe it is)

File: site/docs/components/buttons.md [hunk 2]
```diff
@@ -31,12 +31,30 @@ Primary buttons are are Ignite Red with White text.
 
 ### Sizes
 
-Button sizes have dynamic widths based on a padding-left & padding-right of 24px plus the characters. Heights are set by adding add `.btn`, `.btn-lg` or `.btn-sm`.  Small buttons `.btn-sm` have a height of 32px. Base btns `.btn` have a height of 40px. Large buttons `.btn-lg` have a height of 52px.
+Heights are set by adding add `.btn`, `.btn-lg` or `.btn-sm`.  Small buttons `.btn-sm` have a height of 32px. Base btns `.btn` have a height of 40px. Large buttons `.btn-lg` have a height of 52px. Buttons have min-width of 154px, then have a dynamic width based on character counts and padding. Buttons widths can also be adjusted by adding a `.col-span-#` (# = 1-12) or by adding `.w-100`, to take up 100% of the parent element.
```
Comment (bexux) [inline site/docs/components/buttons.md]: You can remove the second "add" in "Heights are set by adding **add** " and "Buttons widths" - Button can be singular.

File: site/docs/components/buttons.md [hunk 3]
```diff
@@ -54,12 +72,30 @@ Secondary buttons are are White with Ignite Red text.
...
```
Comment (bexux) [inline site/docs/components/buttons.md]: Is this section duplicated from above?  It has the same typos in all three duplicated sizes blocks

File: site/docs/components/buttons.md (hunk 2 above)
Comment (ArianneArcher) [reply site/docs/components/buttons.md]: great catch - thanks

File: scss/_buttons.scss [hunk 4]
```diff
@@ -77,6 +78,15 @@ fieldset:disabled a.btn {
...
 // Custom Flo Primary
+.btn-primary{
+
+  &.disabled,
+  &:disabled {
+    background-color: $gray-400;
+    border-color: $gray-400;
+    opacity: 1;
+  }
+}
```
Comment (bexux) [inline scss/_buttons.scss:92]: I think we also need `fieldset:disabled` which bootstrap has.  https://github.com/twbs/bootstrap/blob/master/scss/_buttons.scss#L44

File: site/docs/components/buttons.md (hunk 3 above)
Comment (ArianneArcher) [reply site/docs/components/buttons.md]: @bexux Just pushed changes, let me know if anything else is needed. :)

File: scss/_buttons.scss (hunk 4 above)
Comment (ArianneArcher) [reply scss/_buttons.scss:92]: Got it

**Review by bexux — APPROVED** [review]

**Pull request merged**